src/test/java/edu/cornell/kfs/concur/ConcurTestConstants.java

```diff
@@ -0,0 +1,49 @@
+package edu.cornell.kfs.concur;
+
+public class ConcurTestConstants {

```

Comment:
I only checked a couple, but if these constants are only used in one fixture, do we need a separate constants est class?